packages/react-reconciler/src/ReactFiber.js

```diff
@@ -310,6 +328,11 @@ export function createWorkInProgress(
   workInProgress.ref = current.ref;
 
   if (enableProfilerTimer) {
+    // We intentionally reset, rather than copy, actualDuration & actualStartTime.
+    // This prevents time from endlessly accumulating in new commits.
+    workInProgress.actualDuration = 0;
+    workInProgress.actualStartTime = 0;
+
```

Comment:
These can be moved to the `else` block above. For the initial clone, these are already 0.

Reply:
True, although I'll have to break this up so that `selfBaseTime` and `treeBaseTime` get copied in both cases. No problem though!